lib/elixir/lib/file.ex

```diff
@@ -886,8 +894,6 @@ defmodule File do
     end
   end
 
-  # src may be a file or a directory, dest is definitely
-  # a directory. Returns nil unless an error is found.
```

Comment:
This comment is wrong, either it belongs to a different function, or it is outdated